src/ui-layout.js

```diff
@@ -41,7 +41,8 @@ angular.module('ui.layout', [])
     opts.sizes = opts.sizes || [];
     opts.maxSizes = opts.maxSizes || [];
     opts.minSizes = opts.minSizes || [];
-    opts.dividerSize = opts.dividerSize || 10; //default divider size set to 10
+
+    opts.dividerSize = opts.dividerSize === undefined ? 10 : opts.dividerSize; //default divider size set to 10
```

Comment:
You are right, not really related. I forgot to take that out. Will PR this separately.